sentinelhub/download/client.py

```diff
@@ -1,13 +1,14 @@
 """
 Module implementing the main download client class
 """
-import concurrent.futures
 import logging
 import warnings
 import os
 import sys
+from concurrent.futures import ThreadPoolExecutor, as_completed
 
 import requests
+from tqdm import tqdm
```

Comment:
In the past it used to be important to do `from tqdm.auto import tqdm` in order to ensure that the progress bar also works in Jupyter notebooks. I don't know if this is still the case.

Reply:
I can just change this since it's a minor change.

Reply:
Yes, I can now confirm there is still this distinction. So we should use `tqdm.auto`.